db/schema.rb

```diff
@@ -68,6 +68,16 @@
 
   add_index "discussions", ["user_id"], name: "index_discussions_on_user_id"
 
+  create_table "external_posts", force: true do |t|
```

Comment:
I think this is old? I don't think we have an `external_posts` table anymore now that posts use STI.